site/content/docs/Repositories/_index.md

```diff
@@ -10,6 +10,7 @@ A repository is a place where Kopia stores its snapshot data. Kopia currently su
 * [Azure Blob Storage](#azure)
 * [Amazon S3](#amazon-s3) (and compatible)
 * [Backblaze B2](#b2)
+* [Google Drive](#google-drive)
```

Comment:
can we clearly mark this as experimental?

Reply:
Done. I added a note under the Google Drive section.